scripts/test-create-project-ci.sh

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+export SKIP_CHANGELOG=true
+export YARN_AUTH_TOKEN="//localhost:4873/:_authToken=token"
+
+yarn verdaccio --config verdaccio.yml & sleep 10
```

Comment:
One way to ensure verdaccio has started, but save spending the full 10 seconds if it was faster, would be to do:

```sh
yarn verdaccio --config verdaccio.yml &
yarn config set registry "http://localhost:4873"
# Wait until Verdaccio has started and is accepting connections.
while ! nc -zw 1 localhost 4873; do sleep 1; done
```

(The wait line deliberately being after the `yarn config set`, since it gives a few more seconds for verdaccio to start, whilst also making use of the time, before sitting around idle)